tests/assets/recipes/baked/daylight-factor.yaml

```diff
@@ -34,25 +34,68 @@ flow:
     description: A grid file
     default: null
     spec: null
+    alias: []
     extensions: null
   - annotations: {}
-    type: DAGFolderInput
-    name: model
-    description: A Honeybee model with radiance properties
+    type: DAGFileInput
+    name: model_hbjson
+    description: Honeybee model as a hbjson file.
     default: null
     spec: null
+    alias:
+    - annotations: {}
+      type: DAGGenericInputAlias
+      name: model
+      description: 'A path to a HBJSON file or a HB model object built with Python
+        or dotnet
+
+        libraries.
+
+        '
+      default: null
+      spec: null
+      platform:
+      - grasshopper
+      handler:
+      - annotations: {}
+        type: IOAliasHandler
+        language: python
+        module: honeybee_rhino.handlers
+        function: hb_model_to_hbjson
+      - annotations: {}
+        type: IOAliasHandler
+        language: csharp
+        module: HBRhino.Model
+        function: HBModelToJSON
+    - annotations: {}
+      type: DAGGenericInputAlias
+      name: model
+      description: null
+      default: null
+      spec: null
+      platform:
+      - rhino
+      handler:
+      - annotations: {}
+        type: IOAliasHandler
+        language: dotnet
```

Comment:
dotnet or csharp?

Reply:
This is a sample. I think csharp is the better choice. I will change this.